javaharness/java/arcs/android/accelerator/PecMessage.kt

```diff
@@ -0,0 +1,48 @@
+package arcs.android.accelerator
+
+import arcs.android.accelerator.Log.logger
+import arcs.api.Constants
+import arcs.api.PecInnerPort
+import com.beust.klaxon.Json
+import com.beust.klaxon.TypeAdapter
+import com.beust.klaxon.TypeFor
+import kotlin.reflect.KClass
+
+/**
+ * Handles all outgoing messages sent by [PecInnerPort.postMessage].
+ */
+class PecMessagePayload(@Json(PecInnerPort.MESSAGE_PEC_ID_FIELD)
+                        val pecId: String,
+                        @Json(Constants.PEC_ENTITY_KEY)
+                        val pecMessage: PecMessage) : ShellMessage(Constants.PEC_MESSAGE) {
```

Comment:
Drive-by nit:

The google/kotlin style for this type of line wrapping looks like:

```kotlin
class PecMessagePayload(
  @Json(PecInnerPort.MESSAGE_PEC_ID_FIELD)
  val pecId: String,
  @Json(Constants.PEC_ENTITY_KEY)
  val pecMessage: PecMessage
) : ShellMessage(Constants.PEC_MESSAGE) {
```